app/routes.py

```diff
@@ -0,0 +1,218 @@
+from app import app, db
+from flask import render_template, flash, redirect, request, url_for, session, make_response
+from app.forms import RegisterForm, LoginForm, AddTaskForm
+from flask_login import current_user, login_user, login_required
+from app.models import Users, TaskList
+from datetime import datetime
+
+@app.route('/')
+@app.route('/home')
+def home():
+	user = {'username': 'rushi'}
+	return render_template('home.html')
+
```

Comment:
What is this route and function???? Remove it.

Reply:
Its the home page
removed this line
user = {'username': 'rushi'}